src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -144,6 +144,8 @@ PAI_TASK_INDEX=$TASK_INDEX
 PAI_CONTAINER_HOST_IP=$CONTAINER_IP
 PAI_CONTAINER_HOST_PORT_LIST=$CONTAINER_PORTS
 PAI_CONTAINER_ID=$CONTAINER_ID
+ENV
+cat <<'ENV' >> $ENV_LIST
 PAI_DEFAULT_FS_URI={{{ hdfsUri }}}
```

Comment:
I think the job data here should be escaped as well. (double curly braces)

Reply:
Since the escape rule in heredoc is different from directly in bash, maybe we should drop the heredoc use here

Reply:
There is another solution, like multipart in chrome, use random string as heredoc boundary.